packages/@react-aria/live-announcer/src/LiveAnnouncer.tsx

```diff
@@ -15,21 +15,20 @@ type Assertiveness = 'assertive' | 'polite';
 /* Inspired by https://github.com/AlmeroSteyn/react-aria-live */
 const LIVEREGION_TIMEOUT_DELAY = 7000;
 
-let liveAnnouncer: LiveAnnouncer | null = null;
-
 /**
  * Announces the message using screen reader technology.
  */
 export function announce(
   message: string,
   assertiveness: Assertiveness = 'assertive',
-  timeout = LIVEREGION_TIMEOUT_DELAY
+  timeout = LIVEREGION_TIMEOUT_DELAY,
+  mode: 'message' | 'ids' = 'message'
```

Comment:
?

Reply:
be able to send in a set of id's for use in `aria-labelledby` which is then live announced
vs sending the message in directly. this allows us to send messages which would otherwise require textContent or some other not friendly way of converting HTML to a string